pkg/arvo/mar/graph/validator/publish.hoon

```diff
@@ -9,8 +9,15 @@
   ++  noun
     |=  p=*
     =/  ip  ;;(indexed-post p)
-    ?>  (gte 2 (lent index.p.ip))
-    ip
+    |^
+    ?+    index.p.ip  !!
+      [@ @ ~]  ip
+    ::  top level post must have title as first content
+        [@ ~]
+      ?>  ?=(^ contents.p.ip)
+      ?>  ?=(%text -.i.contents.p.ip)
```

Comment:
I'm curious: do you think that contents should be more heavily constrained, specifically as [%text %text]~? It is currently not especially constrained: the list can be of any length, and can have any type in the subsequent list items.

Reply:
We have to be able to render the other content types, so I don't think it really makes much of a difference from a development perspective, and it seems unnecessary to limit publish like this. I'd love to see publish as a jupyter style notebook for hoon and I'm sure there will be other usecases that I haven't thought of